consensus/dummy/dynamic_fees_test.go

```diff
@@ -564,3 +565,21 @@ func TestDynamicFeesEtna(t *testing.T) {
 	// lower than the prior base fee minimum.
 	require.Less(nextBaseFee.Int64(), params.ApricotPhase4MinBaseFee)
 }
+
+func TestCalcBaseFeeRegression(t *testing.T) {
+	parentTimestamp := uint64(1)
+	timestamp := parentTimestamp + params.RollupWindow + 1000
+
+	parentHeader := &types.Header{
+		Time:    parentTimestamp,
+		GasUsed: 14_999_999,
+		Number:  big.NewInt(1),
+		BaseFee: big.NewInt(1),
+		Extra:   make([]byte, params.DynamicFeeExtraDataSize),
+	}
+
+	require.Equal(t, int64(1), common.Big1.Int64())
```

Comment:
Both of the checks need to either assert `common.Big1.IsInt64()` or do a direct comparison with `common.Big1.Compare(big.NewInt(1)) == 0`.

If `IsInt64() == false` then the output of `Int64()` is undefined. It's improbable that it would be 1, but not impossible.

Reply:
done, thanks